src/component/pricing/questions.tsx

```diff
@@ -23,16 +23,15 @@ export const questions = [
     content: (
       <>
         <>
-          MT credits are used for Machine Translation providers (such as Google
-          Translate, AWS, DeepL, etc.). One credit ⋍ 1 translated character
-          (except for Tolgee Translator).
+          MT credits are used for Machine Translation providers (such as Tolgee
+          translator, Google Translate, AWS, DeepL, etc.). One credit ⋍ 1
+          translated character.
         </>
         <br />
         <br />
         <>
-          On self-hosted instances, the MT credits only apply to Tolgee AI
-          Translator. For other providers (like Google Translate or Amazon
-          Translate), you provide your own API keys.
+          MT credits renewed every month. Unused credits are not transferred to
+          the next month.
```

Comment:
I'd leave the previous text here as well: On self-hosted instances, the MT credits only apply to Tolgee AI Translator. For other providers (like Google Translate or Amazon Translate), you provide your own API keys.